M2/Macaulay2/packages/A1BrouwerDegrees/Code/ArithmeticMethods.m2

```diff
@@ -212,3 +216,44 @@ solveCongruenceList(List,List) := (L1,L2) -> (
         );
     a
     )
+
+-- Input: two polynomials in one variable
+-- Output: the Sylvester matrix of the two polynomials
+getSylvesterMatrix = method()
+getSylvesterMatrix(RingElement, RingElement) := RingElement => (f, g) -> (
```

Comment:
There's already a `sylvesterMatrix` method function defined in the *Elimination* package.  I'd consider exporting that package and overloading its method instead.  Same with `resultant`.